packages/@react-spectrum/listbox/stories/ListBox.stories.tsx

```diff
@@ -526,6 +527,32 @@ storiesOf('ListBox', module)
     )
   );
 
+storiesOf('ListBox', module)
+  .addDecorator(story => (
+    <div style={{display: 'flex', flexDirection: 'column'}}>
+      <Label id="label">Choose an item</Label>
+      <div style={{display: 'flex', minWidth: '200px', background: 'var(--spectrum-global-color-gray-50)', border: '1px solid lightgray'}}>
+        {story()}
+      </div>
+    </div>
+  ))
+  .add(
+    'async loading, resizable',
+    () => (
+      <div style={{height: '200px', width: '100%', minWidth: '200px', resize: 'both', overflow: 'auto'}}>
```

Comment:
this case where this isn't display flex is the reason i had to include height: inherit in the css again